simapp/sim_test.go

```diff
@@ -905,22 +914,23 @@ func TestAppImportExport(t *testing.T) {
 	ctxA := app.NewContext(true, abci.Header{})
 
 	type StoreKeysPrefixes struct {
+		keyName  string
 		A        sdk.StoreKey
 		B        sdk.StoreKey
 		Prefixes [][]byte
 	}
 
 	storeKeysPrefixes := []StoreKeysPrefixes{
-		{app.keyMain, newApp.keyMain, [][]byte{}},
-		{app.keyAccount, newApp.keyAccount, [][]byte{}},
-		{app.keyStaking, newApp.keyStaking, [][]byte{staking.UnbondingQueueKey,
+		{"main", app.keyMain, newApp.keyMain, [][]byte{}},
+		{"account", app.keyAccount, newApp.keyAccount, [][]byte{}},
+		{"staking", app.keyStaking, newApp.keyStaking, [][]byte{staking.UnbondingQueueKey,
 			staking.RedelegationQueueKey, staking.ValidatorQueueKey}}, // ordering may change but it doesn't matter
-		{app.keySlashing, newApp.keySlashing, [][]byte{}},
-		{app.keyMint, newApp.keyMint, [][]byte{}},
-		{app.keyDistr, newApp.keyDistr, [][]byte{}},
-		{app.keyFeeCollection, newApp.keyFeeCollection, [][]byte{}},
-		{app.keyParams, newApp.keyParams, [][]byte{}},
-		{app.keyGov, newApp.keyGov, [][]byte{}},
+		{"slashing", app.keySlashing, newApp.keySlashing, [][]byte{}},
+		{"mint", app.keyMint, newApp.keyMint, [][]byte{}},
+		{"distribution", app.keyDistr, newApp.keyDistr, [][]byte{}},
```

Comment:
Can we use the ModuleName or each of those? just to not repeat constants along the code

Reply:
++